membership/src/main/scala/zio/membership/NodeState.scala

```diff
@@ -0,0 +1,7 @@
+package zio.membership
+
+final case class NodeState(
+  node: NodeId,
+  incarnation: Int,
```

Comment:
what incarnation mean in this context?

Reply:
This should not be in there at this point. This is relevant for swim suspicion protocol and should move there if we decide to implement it